src/encoded/static/components/body_map.js

```diff
@@ -629,20 +641,21 @@ class BodyMap extends React.Component {
                         />
                     </div>
                     <div className="body-list">
-                        <div className="body-list-inner">
+                        <ul className="body-list-inner">
                             {Object.keys(BodyList).map(b =>
-                                <button
-                                    className={`body-list-element ${((typeof this.state.selectedOrgan === 'string' && this.state.selectedOrgan === b) || (typeof this.state.selectedOrgan !== 'string' && this.state.selectedOrgan.includes(b))) ? 'active' : ''}`}
-                                    id={b}
-                                    onClick={e => this.chooseOrgan(e)}
-                                    onMouseEnter={e => highlightOrgan(e)}
-                                    onMouseLeave={unHighlightOrgan}
-                                    key={b}
-                                >
-                                    {b}
-                                </button>
+                                <li key={b}>
+                                    <button
+                                        className={`body-list-element ${((typeof this.state.selectedOrgan === 'string' && this.state.selectedOrgan === b) || (typeof this.state.selectedOrgan !== 'string' && this.state.selectedOrgan.includes(b))) ? 'active' : ''}`}
```

Comment:
This seems duplicated in line 620. Can this be moved into a method, so changes can take place just once?

Reply:
Excellent point